Plugin~/Src/MeshSync/Include/MeshSync/SceneGraph/msEntityConverter.h

```diff
@@ -1,20 +1,18 @@
 #pragma once
 
 #include "MeshSync/MeshSync.h"
+#include "MeshSync/SceneGraph/msEntity.h"
```

Comment:
What do you think of keeping these as forward declarations in the header files, and putting the include lines on the cpp files ?
I think reducing the header dependencies is better for code understanding and maintenance.

Especially since this header file is "public", meaning that it's also visible from other libraries which depend on MeshSync library (DCC plugins, for example)

Reply:
I meant to revert this change, it was an intermediate solution for an issue I had before. I agree with you comment